NerdyNomads/Text-Savvy
# Added unit tests for App.js and LogoutButton.js

## What does this PR do?
- Set up dependencies for frontend testing
- Added unit tests for `App.js` and `LogoutButton.js`, mainly for actions/components that are affected by Account authentication

### Steps to test/review
1. `cd front-end`
2. `npm install`
3. `npm test` or `npm run test`
4. All tests should pass!

## Checklist before merging
      - Reviewers
      - Assignee
      - Labels
      - Project
      - Milestone
      - Linked Issues
